presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -3783,13 +3783,6 @@ public void testRightJoinWithNullValues()
                 "ON lineitem.orderkey = orders.orderkey");
     }
 
-    @Test
-    public void testJoinWithDuplicateRelations()
-    {
-        assertQuery("SELECT * FROM orders JOIN orders USING (orderkey)", "SELECT * FROM orders o1 JOIN orders o2 ON o1.orderkey = o2.orderkey");
```

Comment:
what is wrong with these tests? It only requires `expected` query to be fixed.

Reply:
There's no easy way to express the same shape without listing all the columns.

It would need to be something like "SELECT o1.orderkey, <all fields in o1 except for orderkey>, <all fields in o2 except orderkey>", so it was just easier to remove it.